tidb-cloud/tidb-cloud-encrypt-cmek.md

```diff
@@ -0,0 +1,125 @@
+
+> **Note:**
+>
+> Currently, this feature is only available upon request. If you need to try out this feature, contact [support](/tidb-cloud/tidb-cloud-support.md).
+
+## Restrictions
+
+- Currently, TiDB Cloud only supports using AWS KMS to provide CMEK.
+- To use CMEK, you need to enable CMEK when creating a project and complete CMEK-related configurations before creating a cluster. You cannot enable CMEK for existing projects.
+- Currently, in CMEK-enabled projects, you can only create [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters hosted on AWS. Dedicated Tier clusters hosted on GCP and [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters are not supported.
+- Currently, for a specific project, you can only enable CMEK for one AWS region. Once you have configured it, you cannot create clusters in other regions within the same project.
+
+## Enable CMEK
+
+If you want to encrypt your data using the KMS owned by your account, take the following steps.
+
+### Step 1. Provision KMS and IAM in your cloud provider
+
+1. Create your CMEK Key on the AWS Key Management Service (KMS) console. Copy the KMS Key ARN. To learn how to create a key, see [Creating Keys](http://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html#create-symmetric-cmk) in the AWS documentation.
+
+2. Create an IAM role and configure the role's access policy to CMEK.
+
+    To make the TiDB cluster function as expected, you need to grant certain permissions to the TiDB account for accessing KMS. Note that this feature is in the development stage and the policy requirement could be subject to change as future features might require more permissions. Here’re the required permissions for supported features at present:
+
+    ```json
+    {
+        "Version": "2012-10-17",
+        "Id": "cmek-policy",
+        "Statement": [
+            // EBS-related policy
+            {
+                "Sid": "Allow access through EBS for all principals in the account that are authorized to use EBS",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "*"
+                },
+                "Action": [
+                    "kms:Encrypt",
+                    "kms:Decrypt",
+                    "kms:ReEncrypt*",
+                    "kms:GenerateDataKey*",
+                    "kms:CreateGrant",
+                    "kms:DescribeKey"
+                ],
+                "Resource": "*",
+                "Condition": {
+                    "StringEquals": {
+                        "kms:CallerAccount": "<pingcap-account>",
+                        "kms:ViaService": "ec2.us-west-2.amazonaws.com"
```

Comment:
The region should not be hard-coded here, user need to specify the region where they want to create cluster in policy:
```suggestion
                        "kms:ViaService": "ec2.<region>.amazonaws.com"
```
If customer don't care about the regions, they can use following condition block:
```json
            "Condition": {
                "StringEquals": {
                    "kms:CallerAccount": "<pingcap-account>"
                },
                "StringLike": {
                    "kms:ViaService": "ec2.*.amazonaws.com"
                }
            }
```